src/main/java/com/parasoft/demoapp/service/GlobalPreferencesService.java

```diff
@@ -295,7 +294,6 @@ public void initializeMqOnStartup(GlobalPreferencesEntity globalPreferences)
             globalPreferencesDto.setOrderServiceSendTo(globalPreferences.getOrderServiceRequestTopic());
             globalPreferencesDto.setOrderServiceListenOn(globalPreferences.getOrderServiceResponseTopic());
             validateMqConfig(globalPreferencesDto);
-            // TODO: initialize Kafka topics on startup
```

Comment:
We nend to change sent to and listen on destinations for Kafka when PDA starts up.